emscripten.py

```diff
@@ -119,7 +119,13 @@ def update_settings_glue(metadata, DEBUG):
   all_exports = metadata['exports'] + list(metadata['namedGlobals'].keys())
   settings.WASM_EXPORTS = [asmjs_mangle(x) for x in all_exports]
 
-  settings.BINARYEN_FEATURES = metadata['features']
+  # start with the MVP features, and add any detected features.
+  settings.BINARYEN_FEATURES = ['--mvp-features'] + metadata['features']
+  if settings.USE_PTHREADS:
+    assert '--enable-threads' in settings.BINARYEN_FEATURES
+  if settings.MEMORY64:
+    assert '--enable-memory64' in settings.BINARYEN_FEATURES
```

Comment:
If we don't think these are (or should be) necessary, it would be good to add a TODO about that.

Reply:
Yeah I mostly added this so the CI could prove they never fired.  I agree they are probably unnecessary.